crates/router/src/types/api.rs

```diff
@@ -131,7 +142,8 @@ impl<
             + Dispute
             + FileUpload
             + ConnectorTransactionId
-            + Payouts,
+            + Payouts
+            + ConnectorErrorTypeMapping,
```

Comment:
Do we need to enforce this for every connector? This is not necessary for all the connectors right?

Reply:
Yeah this is optional and need to be implemented only when the utils function is called where connector sends list of errors